### DISCUSSION TITLE
Location of index.redb on Mac ?

### DISCUSSION THREAD

**Discussion by BitcoinWay** [ordinals/ord#2856] [Q&A] [answered]
I'm getting the following error:
error: failed to open index: DB corrupted: Expected file format version 1, found 109
I'm trying to rebuild the index but can't find it on a a Mac. Anyone know the location ?
Thanks!

**Comment by so7ow** [comment, marked as answer]
`~/Library/Application Support/ord`

**Reply by BitcoinWay** [reply]
there isn't an ord directory in Application Support. I'm able to run ord and get a version ( the latest ) however when I run wallet balance I get the error posted above

**Reply by so7ow** [reply]
You're probably looking in `/Library` instead of `~/Library`.

**Reply by so7ow** [reply]
(The ~ is a shortcut way of noting /Users/*username*)

**Reply by BitcoinWay** [reply]
beautiful! thanks, that's it
